ui-tests/src/test/java/org/openmrs/reference/page/AttachmentsPage.java

```diff
@@ -0,0 +1,48 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class AttachmentsPage extends Page {
+
```

Comment:
this is interesting, you are making me think of moving all our pages out of qaframework into distro. i will review this shortlyk

Reply:
ThoughtWorks
I think basing on our qa values, its crucial our technical work to align with the software standards. Looking at this from a technical angle, pages are resources to the definition-Steps; so accessing these resources would be better through a library.

Reply:
i already moved out the pages we had in qaframework and thus we will be keeping all future pages in here